api/hasura/actions/_handlers/allocationCsv.ts

```diff
@@ -35,15 +35,9 @@ async function handler(req: VercelRequest, res: VercelResponse) {
   const grant = payload.grant ?? epochObj.grant;
 
   const totalTokensSent = epochObj.token_gifts.length
-    ? epochObj.token_gifts
-        .map(g => g.tokens)
-        .reduce((total, tokens) => tokens + total)
+    ? epochObj.token_gifts.reduce((total, { tokens }) => total + tokens, 0)
     : 0;
-  const circle = await getCircleDetails(
-    circle_id,
-    epochObj.id,
-    epochObj.end_date
```

Comment:
End date no longer needed as it's irrelevant when the user was deleted, we don't want to account their GIVE if they are deleted, even if after the epoch has ended